cdbot/cogs/cyber.py

```diff
@@ -353,10 +353,10 @@ def __init__(self, bot: Bot):
         Gets the number of 100% and true 100% users
         """
 
-        game_r = utils.get(ctx.guild.roles, id=HUNDRED_ROLE_ID)
-        true_r = utils.get(ctx.guild.roles, id=TRUE_HUNDRED_ROLE_ID)
+        game_r = utils.get(ctx.guild.roles, id=HUNDRED_PERCENT_ROLE_ID)
+        true_r = utils.get(ctx.guild.roles, id=TRUE_HUNDRED_PERCENT_ROLE_ID)
 
-        await ctx.send((f"There are {len(game_r.members)} 100% users, of which {len(true_r.members)} have also "
+        await ctx.send((f"There are {len(game_r.members)} that have completed Cyberstart Game. Out of them, {len(true_r.members)} have also "
```

Comment:
This wraps over the 120-char that was being used to wrap the original line. Probably worth re-wrapping.